### PR TITLE
docs: add passkeys

### PR DESCRIPTION
**Pull request by tomekpapiernik**
This adds ~~Apple~~ Passkeys to the passwordless documentation. The thing "just works" so there's not much more to add. Happy to hear your suggestions if you see it differently.

## Related Issue or Design Document

https://github.com/ory/docs/issues/949

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: src/sidebar.js [hunk 1]
```diff
@@ -163,7 +163,12 @@ module.exports = {
         {
           type: "category",
           label: "Passwordless",
-          items: ["kratos/passwordless/webauthn"],
+          items: [
+            {
+              type: "autogenerated",
+              dirName: "kratos/passwordless",
```
Comment (kmherrmann) [inline src/sidebar.js:169]: I propose to:
- move Passwordless above Social Sign-In?
- rename Apple Passkeys to "Passkeys & Biometric" and move above WebauthN

File: docs/kratos/passwordless/05_apple-passkeys.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<BrowserWindow url="https://my.app/login">
+
+![Apple Passkeys in Safari](../_static/webauthn/2.png)
+
+</BrowserWindow>
+```
+
+:::tip
+
+To learn more about Apple Passkeys, read [this Apple support article](https://support.apple.com/en-us/HT213305).
```
Comment (kmherrmann) [inline docs/kratos/passwordless/05_apple-passkeys.mdx:43]: Google link: https://developers.google.com/identity/passkeys

File: docs/kratos/passwordless/05_apple-passkeys.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+id: apple-passkeys
+title: Passwordless sign-in and sign-up with Apple Passkeys
+sidebar_label: Apple Passkeys
+---
+
+# Apple Passkeys
```
Comment (kmherrmann) [inline docs/kratos/passwordless/05_apple-passkeys.mdx:7]: Passkeys are supported and pushed by Apple, Google and Microsoft, the 3 most relevant platform vendors, and thus are about to become ubiquitous. We should make that clear!

For SEO purposes, we want "Apple Passkeys" on the page as a phrase, too, though.

**Review by kmherrmann — CHANGES_REQUESTED** [review]
Let's make this not only apple-specific. Otherwise looks great, thanks!

File: src/sidebar.js (hunk 1 above)
Comment (tomekpapiernik) [reply src/sidebar.js:169]: Sure, I can move it above social sign-in. I don't think "Passkeys & Biometric" is a good name though. When using passkeys, any biometrics that the user has to "supply" are required to confirm that they want to use the passkey. The passkey itself is just cryptography.

Comment (tomekpapiernik) [reply src/sidebar.js:169]: Also, passkeys don't require biometrics. If a device is not biometrics-enabled, users can still use passkeys.

**Review by piotrmsc — APPROVED** [review]

**Pull request merged**